src/framework/app-options.js

```diff
@@ -71,49 +71,49 @@ class AppOptions {
     /**
      * The sound manager
      *
-     * @type {SoundManager}
+     * @type {SoundManager.constructor}
      */
     soundManager;
 
     /**
      * The graphics device.
      *
-     * @type {GraphicsDevice}
+     * @type {GraphicsDevice.constructor}
```

Comment:
this one we pass an instance, and possibly others, not all we pass as types.

Reply:
ahh yes of course, updated.